routes/api.php

```diff
@@ -30,12 +29,18 @@
     if ($request->all()['action'] !== 'created') {
         return;
     }
-    Artisan::call('gh:change-file', [
-        'commentContent' => $request->all()['comment']['body'],
-        'filePath' => $request->all()['comment']['path'],
-        'lineNumber' => $request->all()['comment']['line'],
-        'repository' => $request->all()['pull_request']['head']['repo']['name'],
-        'owner' => $request->all()['pull_request']['head']['repo']['owner']['login'],
-        'branch' => $request->all()['pull_request']['head']['ref'],
-    ]);
+    if (explode(' ', $request->all()['comment']['body'])[0] !== '/gen') {
+        logger('aaa');
+
+        return;
+    }
+    \App\Actions\ChangeFileWithGhComment::run(
+        $request->all()['comment']['body'],
+        $request->all()['comment']['path'],
+        $request->all()['comment']['line'],
+        $request->all()['pull_request']['head']['repo']['name'],
+        $request->all()['pull_request']['head']['repo']['owner']['login'],
+        $request->all()['pull_request']['head']['ref'],
+        $request->all()['comment']['start_line'],
+    );
```

Comment:
/gen passing $request->all()['comments']['body'] instead of $request->all()['comment']['body'] as first param